Formula/watchman.rb

```diff
@@ -23,7 +29,8 @@ def install
                           "--with-pcre",
                           # we'll do the homebrew specific python
                           # installation below
-                          "--without-python"
+                          "--without-python",
+                          "--enable-statedir=#{var}/run/watchman"
```

Comment:
I agree. Bottles should preserve the fidelity of the permissions of what's installed (of course, including whatever permissions "fixes" brew may have done during install).